build-scripts/setup.sh

```diff
@@ -0,0 +1,297 @@
+#!/bin/bash
+set -e -o pipefail
+
+THISDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+echo "Working in `pwd`"
+source ${THISDIR}/functions.sh
+
+if [[ "x$REPO_NAME" == "x" ]]; then
+    REPO_NAME=kontext
+fi
+
+THISSCRIPT=`basename "$0"`
+minisep "$THISSCRIPT ($REPO_NAME)" 
+start=`date +%s`
+
+# =========
+# env
+
+MANATEE_FROM_PACKAGES=false
+MANATEE_OS_VERSION=1604
+
+export FS=${THISDIR}/..
+
+# allow multiple instances
+if [[ "x$KONTEXT_PREFIX" == "x" ]]; then
+    export KONTEXT_PREFIX=/opt/kontext
+fi
+
+export DEPSDIR=${KONTEXT_PREFIX}/bits
+export DATADIR=${KONTEXT_PREFIX}/data
+export CACHEDIR=${KONTEXT_PREFIX}/cache
+
+# allow having source code somewhere else
+if [[ "x$KONTEXTDIR" == "x" ]]; then
+    export KONTEXTDIR=${KONTEXT_PREFIX}/installation
+fi
+
+export THEMEDIR=${KONTEXTDIR}/public/files/themes
+export CONFIGDIR=${KONTEXTDIR}/conf
+
+export DEPS_PREFIX=${KONTEXT_PREFIX}
+export DEPSLIBDIR=${DEPS_PREFIX}/lib
+export DEPSINCLUDEDIR=${DEPS_PREFIX}/include
+
+if [[ "x$PORT" == "x" ]]; then
+    export PORT=5000
+fi
+
+if [[ "x$MYSQLUSER" == "x" ]]; then
+    export MYSQLUSER=root
+fi
+if [[ "x$MYSQLPASS" == "x" ]]; then
+    export MYSQLPASS=
+fi
+
+mkdir -p ${DEPSDIR}
+sudo chown -R ${USER}:${USER} /opt/kontext
```

Comment:
I guess this and any other instance of `/opt/kontext` should be `${KONTEXTDIR}` or some other variable